extensions/txAdminClient/resource/cl_scaleform.lua

```diff
@@ -0,0 +1,89 @@
+-- Announce Scaleform to every players
+local announce_title = "Staff announce"  -- Set the title of the announcement here.
+local AnnouncementScaleform
+Scaleform = {}
+local scaleform = {}
+scaleform.__index = scaleform
+
+
+function StartDraw(time)
+	Citizen.CreateThread(function()
+		local fading = false
+		while not AnnouncementScaleform do Wait(0) end
+		local StartTime = GetGameTimer()
+		while true do
+			AnnouncementScaleform:Draw2D()
+
+			if GetGameTimer() - StartTime > time * 1000 then 
+					fading = true
+					AnnouncementScaleform:CallFunction("SHARD_ANIM_OUT", 2, 1) 
+			end
+			if GetGameTimer() - StartTime > (time + 1.25) * 1000  then
+					AnnouncementScaleform:Dispose()
+					fading = false
+				return 
+			end
+			Wait(0)
+		end
+	end)
+end
+
+RegisterNetEvent("Scaleform:Announce")
```

Comment:
Not sure about using an event for this instead of a function call or export.  
Check how they work to make sure no one else can trigger this other than txAdmin.  
  
Also, I don't want to use the default event name for scaleform to prevent any conflict issues.

Reply:
As first comment says, i didnt created it all, but it looks ok for me.

Reply:
we can't have it as `Scaleform:Announce`, since the original scaleform code already uses it.